### PR TITLE
Integration branch | Story list pages and tags

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Story filter pages
- Add tags to story cards and story heroes
- Add architectural decision records

# Review By (Date)
- July 11 EOD

# Criticality
- 7

# Review Tasks

## Setup tasks and/or behavior to test

1. All the PRs that were merged into this branch has been reviewed.
2. Note: Release won't go out until we publish all the story filter pages in Storyblok.

### CODE REVIEW

**Comment by sherakama** [comment]
> @sherakama I create this branch/PR thinking we can merge any smaller PRs related to the story list pages into this branch, so we can get them bundled at the end and get a single merge into `dev` and then production when we're ready - LMK what you think.

Perfect. Integration branches are great. You can also set up a 'branch build' in netlify to have a dedicated URL for the review work.

**Comment by yvonnetangsu** [comment]
> > @sherakama I create this branch/PR thinking we can merge any smaller PRs related to the story list pages into this branch, so we can get them bundled at the end and get a single merge into `dev` and then production when we're ready - LMK what you think.
> 
> Perfect. Integration branches are great. You can also set up a 'branch build' in netlify to have a dedicated URL for the review work.

Thanks for the confirmation, @sherakama ! branch build has been set up 🤠 
https://story-list-pages--giving-campaign.netlify.app/

**Comment by yvonnetangsu** [comment]
@sherakama ticket https://stanfordits.atlassian.net/browse/GIVCAMP-358 is created for you for when I'm on vacation. Thank you in advance 😸  Please base off of this branch.

File: components/Storyblok/SbStoryFilterPage/index.ts
Comment (sherakama) [inline components/Storyblok/SbStoryFilterPage/index.ts:1]: This file is missing the exports.

**Review by sherakama — APPROVED** [review]
GTG.

Please fix that one empty file.

File: components/Storyblok/SbStoryFilterPage/index.ts
Comment (yvonnetangsu) [reply components/Storyblok/SbStoryFilterPage/index.ts:1]: Thanks @sherakama - I added to the index file and update the import in StoryblokProvider so it's consistent with our convention in this repo.

**Comment by yvonnetangsu** [comment]
Merging this into `dev` - thanks @sherakama!

**Comment by yvonnetangsu** [comment]
@sherakama Should I use regular merge commit or squash merge to merge this integration branch into `dev`?

**Pull request merged**